src/core/qgsvectorlayer.h

```diff
@@ -786,6 +786,11 @@ class CORE_EXPORT QgsVectorLayer : public QgsMapLayer, public QgsExpressionConte
     virtual QString getStyleFromDatabase( const QString& styleId, QString &msgError );
 
     /**
+     * Will delete the named style corresponding to style id provided from the database
```

Comment:
Can you add a "@note added in QGIS 3.0" to all newly added public/protected members? (This helps plugin devs know what API calls they can safely use)